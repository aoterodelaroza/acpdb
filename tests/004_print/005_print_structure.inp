## check: 005_print_structure.out -a1e-10
## labels: regression quick

verbose
connect ../zz_source/bsip2-small/bsip2.db
print structure
