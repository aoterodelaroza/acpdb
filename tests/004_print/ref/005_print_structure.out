%% verbose
%% connect ../zz_source/bsip2-small/bsip2.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

%% print structure
* PRINT: print the contents of the database 

| id| key| set| ismolecule| charge| multiplicity| nat|
| 1| s22.nh3_nh3| 1| 1| 0| 1| 8|
| 2| s22.nh3_nh3_1| 1| 1| 0| 1| 4|
| 3| s22.nh3_nh3_2| 1| 1| 0| 1| 4|
| 4| s22.h2o_h2o| 1| 1| 0| 1| 6|
| 5| s22.h2o_h2o_1| 1| 1| 0| 1| 3|
| 6| s22.h2o_h2o_2| 1| 1| 0| 1| 3|
| 7| s22.h2co2_h2co2| 1| 1| 0| 1| 10|
| 8| s22.h2co2_h2co2_1| 1| 1| 0| 1| 5|
| 9| s22.h2co2_h2co2_2| 1| 1| 0| 1| 5|
| 10| s22.formamide_formamide| 1| 1| 0| 1| 12|
| 11| s22.formamide_formamide_1| 1| 1| 0| 1| 6|
| 12| s22.formamide_formamide_2| 1| 1| 0| 1| 6|
| 13| s22.uracil_uracil_hb| 1| 1| 0| 1| 24|
| 14| s22.uracil_uracil_hb_1| 1| 1| 0| 1| 12|
| 15| s22.uracil_uracil_hb_2| 1| 1| 0| 1| 12|
| 16| s22.pyridoxine_aminopyridine| 1| 1| 0| 1| 25|
| 17| s22.pyridoxine_aminopyridine_1| 1| 1| 0| 1| 12|
| 18| s22.pyridoxine_aminopyridine_2| 1| 1| 0| 1| 13|
| 19| s22.adenine_thymine_wcc1| 1| 1| 0| 1| 30|
| 20| s22.adenine_thymine_wcc1_1| 1| 1| 0| 1| 15|
| 21| s22.adenine_thymine_wcc1_2| 1| 1| 0| 1| 15|
| 22| s22.ch4_ch4| 1| 1| 0| 1| 10|
| 23| s22.ch4_ch4_1| 1| 1| 0| 1| 5|
| 24| s22.ch4_ch4_2| 1| 1| 0| 1| 5|
| 25| s22.c2h4_c2h4| 1| 1| 0| 1| 12|
| 26| s22.c2h4_c2h4_1| 1| 1| 0| 1| 6|
| 27| s22.c2h4_c2h4_2| 1| 1| 0| 1| 6|
| 28| s22.c6h6_ch4| 1| 1| 0| 1| 17|
| 29| s22.c6h6_ch4_1| 1| 1| 0| 1| 12|
| 30| s22.c6h6_ch4_2| 1| 1| 0| 1| 5|
| 31| s22.c6h6_c6h6_pd| 1| 1| 0| 1| 24|
| 32| s22.c6h6_c6h6_pd_1| 1| 1| 0| 1| 12|
| 33| s22.c6h6_c6h6_pd_2| 1| 1| 0| 1| 12|
| 34| s22.pyrazine_pyrazine| 1| 1| 0| 1| 20|
| 35| s22.pyrazine_pyrazine_1| 1| 1| 0| 1| 10|
| 36| s22.pyrazine_pyrazine_2| 1| 1| 0| 1| 10|
| 37| s22.uracil_uracil_stack| 1| 1| 0| 1| 24|
| 38| s22.uracil_uracil_stack_1| 1| 1| 0| 1| 12|
| 39| s22.uracil_uracil_stack_2| 1| 1| 0| 1| 12|
| 40| s22.indole_c6h6_stack| 1| 1| 0| 1| 28|
| 41| s22.indole_c6h6_stack_1| 1| 1| 0| 1| 16|
| 42| s22.indole_c6h6_stack_2| 1| 1| 0| 1| 12|
| 43| s22.adenine_thymine_stack| 1| 1| 0| 1| 30|
| 44| s22.adenine_thymine_stack_1| 1| 1| 0| 1| 15|
| 45| s22.adenine_thymine_stack_2| 1| 1| 0| 1| 15|
| 46| s22.c2h4_c2h2| 1| 1| 0| 1| 10|
| 47| s22.c2h4_c2h2_1| 1| 1| 0| 1| 6|
| 48| s22.c2h4_c2h2_2| 1| 1| 0| 1| 4|
| 49| s22.c6h6_h2o| 1| 1| 0| 1| 15|
| 50| s22.c6h6_h2o_1| 1| 1| 0| 1| 12|
| 51| s22.c6h6_h2o_2| 1| 1| 0| 1| 3|
| 52| s22.c6h6_nh3| 1| 1| 0| 1| 16|
| 53| s22.c6h6_nh3_1| 1| 1| 0| 1| 12|
| 54| s22.c6h6_nh3_2| 1| 1| 0| 1| 4|
| 55| s22.c6h6_hcn| 1| 1| 0| 1| 15|
| 56| s22.c6h6_hcn_1| 1| 1| 0| 1| 12|
| 57| s22.c6h6_hcn_2| 1| 1| 0| 1| 3|
| 58| s22.c6h6_c6h6_t| 1| 1| 0| 1| 24|
| 59| s22.c6h6_c6h6_t_1| 1| 1| 0| 1| 12|
| 60| s22.c6h6_c6h6_t_2| 1| 1| 0| 1| 12|
| 61| s22.indole_c6h6_t| 1| 1| 0| 1| 28|
| 62| s22.indole_c6h6_t_1| 1| 1| 0| 1| 16|
| 63| s22.indole_c6h6_t_2| 1| 1| 0| 1| 12|
| 64| s22.phenol_phenol| 1| 1| 0| 1| 26|
| 65| s22.phenol_phenol_1| 1| 1| 0| 1| 13|
| 66| s22.phenol_phenol_2| 1| 1| 0| 1| 13|

